bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/thing/channelTypes.xml

```diff
@@ -400,26 +496,42 @@
 		<label>@text/channel-type.mielecloud.light_switch.label</label>
 		<description>@text/channel-type.mielecloud.light_switch.description</description>
 		<category>Switch</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Light</tag>
+		</tags>
 	</channel-type>
 
 	<channel-type id="light_can_be_controlled">
 		<item-type>Switch</item-type>
 		<label>@text/channel-type.mielecloud.light_can_be_controlled.label</label>
 		<description>@text/channel-type.mielecloud.light_can_be_controlled.description</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Light</tag>
```

Comment:
This is more like metadata: "Indicates if the light of the device can be controlled.". I'd propose to remove the tags.